client/src/components/overview/GalleryView.jsx

```diff
@@ -179,18 +199,36 @@ function Overview(props) {
       });
     }
   });
-
+  /* find(obj => obj.style_id === currentStyleId) */
   if (!expanded) {
     return (
       <StyledOverviewGrid>
```

Comment:
This code looks much cleaner than before, I should look to do something similar.